src/textual/geometry.py

```diff
@@ -1,21 +1,22 @@
 from __future__ import annotations
 
-from typing import Any, NamedTuple, TypeVar
+from typing import Any, NamedTuple, TypeVar, Iterable
 
 
-T = TypeVar("T", int, float)
+Number = TypeVar("Number", int, float)
```

Comment:
I applaud the effort to make the code self-documenting, but while it's true these are numbers now, they might not *always* be numbers.

I'd prefer to defer that decision to later.